co-log/tutorials/1-intro/Intro.md

```diff
@@ -1,37 +1,37 @@
 # Intro: Using `LogAction`
 
 This tutorial is an introduction to `co-log`. It contains basic examples of
-using core data types and functions.
+using `co-log`'s core data types and functions.
 
-You can run this tutorial by calling the following command:
+You can run this tutorial by executing the following command:
 
 ```shell
 cabal new-run tutorial-intro
 ```
 
 ## Preamble: imports and language extensions
 
-Since this is a literate haskell file, we need to specify all our language
-extensions and imports up front.
+Since this is a literate Haskell file, we need to specify all our imports up
+front:
 
-```haskell
+```idris
 import Colog.Core (LogAction (..), (<&), logStringStdout)
 ```
 
 ## Core data type
 
 `co-log` is based on the following data type:
 
-```idris
+```haskell
```

Comment:
This snippet should be `idris` since it's not part of the code that should be compiled. It's just the definition of `LogAction` from `co-log-core`